# HG changeset patch
# User Dan Villiom Podlaski Christiansen <danchr@gmail.com>
# Date 1666963816 -7200
#      Fri Oct 28 15:30:16 2022 +0200
# Branch 1.0.x
# Node ID 7c80c67f737f0e0eeee275deed0ec53e055207a3
# Parent  fd5ee816fe725f0a8fa334efa6737165c834c766
ci: work around Git security fixes that broke our tests

So, it seems[1] that Git now treats local repositories as unsafe for
submodules due to some oddity with hard links and protecting e.g.
`~/.git`. Such attacks seem unlikely to pass through code review, so
let's just allow local submodules for testing.

[1] https://lore.kernel.org/git/xmqq4jw1uku5.fsf@gitster.g/T/#u

diff --git a/tests/testutil b/tests/testutil
--- a/tests/testutil
+++ b/tests/testutil
@@ -29,6 +29,8 @@
 cat >> $TESTTMP/.gitconfig <<EOF
 [init]
 defaultBranch = master
+[protocol.file]
+allow = always
 EOF
 
 # Standard checks for external dependencies
